ingest/expression_files/mtx.py

```diff
@@ -132,14 +144,14 @@ def is_sorted(idx: int, visited_expression_idx: List[int]):
 
     def execute_ingest(self):
```

Comment:
A `"""`-formatted docstring would be nice here, so maintainers can hover over calls to this and see a summary.